model/addons_mgmt/v1/addon_status_type.model

```diff
@@ -0,0 +1,25 @@
+/*
+Copyright (c) 2022 Red Hat, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+
+// Representation of an addon status.
+class AddonStatus {
+    // Identifier for co-relating current AddonCR revision and reported status.
```

Comment:
We are missing the addon_id from here 

https://gitlab.cee.redhat.com/ocm/ocm-addons-service/-/blob/main/openapi/ocm-addons-service.yaml#L769

Reply:
looks like that is all that is missing, i am able to create, get, list and delete using the SDK when I added the addon_id